tiledb/sm/cpp_api/profile_experimental.h

```diff
@@ -216,6 +211,19 @@ class Profile {
     }
   }
 
+  /** Loads the profile from the local file. */
+  void load() {
```

Comment:
Perhaps this should be a factory function? so users can do `Profile::load(name, profile_dir)`